.gitignore

```diff
@@ -12,3 +12,5 @@ inst/doc
 docs/*
 doc
 Meta
+data/nba_elo.rda
+man/nbaallelo.Rd
```

Comment:
Please remove these two lines from `.gitignore`. We don't want `git` to ignore these files.